### PR TITLE
Created INodeStop & stops and crossroads now have radius

### LINKED ISSUE #166: Left click node part 1
* clicking node highlights it
* highlight with fillcolor + circle with opacity
type of node determines the fill & circle color
* circle has a default radius however, if a node has radius, override circle's default radius

part 2: #156 

- Started on https://github.com/HSLdevcom/jore-map-ui/compare/166-node-highlightning, can be continued.

### PR DESCRIPTION
**Pull request by Divinare**
Closes #166 

Changed styling of nodes on map.
Added circle to also selected crossroads.

### CODE REVIEW

File: src/components/map/layers/markerLayer.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import '../../shared/styles/common.scss';
```
Comment (kronken) [inline src/components/map/layers/markerLayer.scss]: unnecessary import

File: src/components/map/layers/NodeLayer.tsx [hunk 2]
```diff
@@ -17,37 +17,82 @@ interface MarkerLayerProps {
...
+        const borderColor = isSelected ? '#727272' : '#727272';
+        const fillColor = isSelected ? '#727272' : '#FFF';
+        return `<div
+            style="border-color: ${borderColor}; background-color: ${fillColor}"
+            class=${s.nodeContent}
+        />`;
+    }
+
+    private getNodeStopMarkerHtml = (isSelected: boolean) => {
+        const borderColor = isSelected ? '#007ac9' : '#007ac9';
```
Comment (kronken) [inline src/components/map/layers/NodeLayer.tsx]: This gives same result independent on if isSelected or not

File: src/components/map/layers/NodeLayer.tsx [hunk 3]
```diff
@@ -17,37 +17,82 @@ interface MarkerLayerProps {
...
-    SELECTED_FILL_COLOR = '#f44268',
-}
+const DEFAULT_RADIUS = 25;
 
 @inject('popupStore', 'toolbarStore', 'sidebarStore')
 @observer
 export default class NodeLayer extends Component<MarkerLayerProps> {
-    private getNodeMarkerHtml = (borderColor: string, fillColor: string) => {
+    private getNodeCrossroadMarkerHtml = (isSelected: boolean) => {
+        const borderColor = isSelected ? '#727272' : '#727272';
```
Comment (kronken) [inline src/components/map/layers/NodeLayer.tsx]: This gives same result independent on if isSelected or not

File: src/components/map/layers/NodeLayer.tsx [hunk 4]
```diff
@@ -17,37 +17,82 @@ interface MarkerLayerProps {
     sidebarStore?: SidebarStore;
 }
 
-enum color {
-    STOP_BORDER_COLOR = '#f17c44',
-    CROSSROAD_BORDER_COLOR = '#666666',
-    NORMAL_FILL_COLOR = '#FFF',
-    SELECTED_FILL_COLOR = '#f44268',
-}
```
Comment (kronken) [inline src/components/map/layers/NodeLayer.tsx]: Any reason why removing these and setting the color codes directly in the code? I think this old style to specify colors was rather ok.

**Comment by Divinare** [comment]
You can review this again.

**Pull request merged**